modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -49,6 +49,17 @@ include::design-center::partial$design-add-ruleset-dependencies.adoc[]
 
 Even if your projects have not been identified as requiring action, you can proactively check the conformance of your APIs by adding published rulesets as dependencies to your API project. 
 
+When your API fails conformance to rulesets your organization validates against, you get an email notification. 
```

Comment:
Suggestion: If your API fails to conform to the rulesets that your organization validates against, you get an email notification.

("If" instead of "When" because "When" implies that this always happens. "Conformance" is correct, but it is not a common word.)